plugins/circleci/src/components/BuildsPage/lib/CITable/CITable.tsx

```diff
@@ -212,6 +212,14 @@ const generatedColumns: TableColumn[] = [
   {
     title: 'Workflow',
     field: 'workflow.name',
+    highlight: true,
+    render: (row: Partial<CITableBuildInfo>) => (
+      <Link
+        to={`https://app.circleci.com/pipelines/workflows/${row?.workflow?.id}`}
```

Comment:
this is a nice simple fix!

Note: this workflow link due to its hardcoding will ONLY work on CircleCI Cloud SaaS Service, but not to anyone hosting their own solution on their own URL.  Sounds like Mark you use Cloud offering (as do I), so may be something we add to the issues log for a `help wanted` to have someone help refactor to pull in the host from the `app-config.yaml`.  (i know i can't quite commit to that myself this week!)

Reply:
I mean the ideal fix would be that the `row?.workflow?.url` return a URL that is able to link to a workflow. As right now it returns `https://circleci.com/gh/myorg/myrepo/3074/workflows/85b84cac-861d-48b6-82c7-9081fb8d6e44` like you said in the comment above.

Reply:
I've actually been dialoging with staff from CircleCI about a TypeScript-compatible SDK, and posted on their discuss forum about it: https://discuss.circleci.com/t/any-official-plans-for-a-circleci-api-sdk/45822

Indeed, might need a change to that source repo but it seems it's gone a bit dormant, but might not hurt to try!

I think any further dialog maybe we pivot into #16193 and that way anyone who can/wants to pick it up may have the reference to that change.

Otherwise, indeed THIS PR would enhance it for my team :)